pkgs/top-level/all-packages.nix

```diff
@@ -21360,14 +21331,7 @@ in
 
   gpx = callPackage ../applications/misc/gpx { };
 
-  gqrx = libsForQt514.callPackage ../applications/radio/gqrx {
-    gnuradio = gnuradio3_7Minimal;
-    # Use the same gnuradio for gr-osmosdr as well
-    gr-osmosdr = gr-osmosdr.override {
-      gnuradio = gnuradio3_7Minimal;
-      pythonSupport = false;
-    };
-  };
+  gqrx = gnuradioMinimal.callPackage ../applications/radio/gqrx { };
```

Comment:
I think we should avoid using the `callPackage` from another scope, and instead pass in that scope.